integrationtest/mapper/orm-coordination-outbox-polling/src/test/java/org/hibernate/search/integrationtest/mapper/orm/coordination/outboxpolling/automaticindexing/OutboxPollingAutomaticIndexingEventSendingIT.java

```diff
@@ -78,6 +78,8 @@ public void setup(OrmSetupHelper.SetupContext setupContext, ReusableOrmSetupHold
 		);
 
 		setupContext.withProperty( "hibernate.search.coordination.outbox_event_finder.provider", outboxEventFinder.provider() )
+				// use timebase uuids to get predictable sorting order
+				.withProperty( "hibernate.search.coordination.entity.mapping.outboxevent.uuid_gen_strategy", "time" )
```

Comment:
well well, if that isn't cheating :)

Ok, makes sense!

Reply:
😊

> Let's run tests multiple times on each database before we merge this, because we might have unpleasant surprises (flaky tests).

Because we did get the flaky tests 😉